eq-author/src/components/QuestionPicker/index.js

```diff
@@ -0,0 +1,282 @@
+import React, { useState, useEffect, useContext } from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { getPages } from "utils/questionnaireUtils";
+
+import { colors } from "constants/theme";
+
+import { ReactComponent as WarningIcon } from "assets/icon-warning-round.svg";
+import { ReactComponent as FolderIcon } from "assets/icon-folder.svg";
+
+import SelectedPageContext, {
+  SelectedPagesProvider,
+} from "./SelectedPagesContext";
+
+import Modal from "components/modals/Modal";
+import SearchBar from "components/SearchBar";
+import IconText from "components/IconText";
+import Button from "components/buttons/Button";
+import ButtonGroup from "components/buttons/ButtonGroup";
+import ScrollPane from "components/ScrollPane";
+import NoSearchResults from "components/NoSearchResults";
+
+import Item from "./Item";
+import List from "./List";
+
+const StyledModal = styled(Modal)`
+  .Modal {
+    padding: 0;
+    padding-top: 1em;
+    width: 45em;
+  }
+`;
+
+const Header = styled.header`
+  margin: 0 1.5em;
+
+  > * {
+    margin-bottom: 0.5em;
+  }
+`;
+
+const Main = styled.main`
+  overflow: hidden;
+  height: 25em;
+`;
+
+const Footer = styled.footer`
+  padding: 1.5em;
+`;
+
+const Title = styled.h2`
+  font-weight: bold;
+  font-size: 1.2em;
+  color: ${colors.text};
+  margin-bottom: 0.75em;
+`;
+
+const WarningPanel = styled(IconText)``;
+
+const isSelected = (selectedItemsArr, item) =>
+  selectedItemsArr.findIndex(
+    (selectedAnswer) => selectedAnswer.id === item.id
+  ) !== -1;
+
+const Page = ({ page }) => {
+  const { title, displayName, alias } = page;
+  const { selectedPages, updateSelectedPages } =
+    useContext(SelectedPageContext);
+
+  const itemSelected = isSelected(selectedPages, page);
+
+  const handleClick = () => {
+    if (itemSelected) {
+      const selectionWithoutThisPage = selectedPages.filter(
+        (selectedPage) => selectedPage.id !== page.id
+      );
+      updateSelectedPages(selectionWithoutThisPage);
+    } else {
+      updateSelectedPages([...selectedPages, page]);
+    }
+  };
+  return (
+    <Item
+      title={title.replace(/(<([^>]+)>)/gi, "") || displayName}
```

Comment:
There's a `stripHTML` function exported from `eq-author/src/utils/stripHTML.js` - would that work here instead of the regex?